### PR TITLE
LFR-1915 Adding Flow types to Card components

### PR DESCRIPTION
**Pull request by lukeamarsh**
Adding flow types to all card components

### CODE REVIEW

File: packages/core/src/components/Cards/CondensedSpaceCard/CondensedSpaceCard.js [hunk 1]
```diff
@@ -1,13 +1,24 @@
-import PropTypes from 'prop-types';
-import React from 'react';
+//@flow
+import * as React from 'react';
```
Comment (dbatten5) [inline packages/core/src/components/Cards/CondensedSpaceCard/CondensedSpaceCard.js]: do you need to `import *` here?

File: packages/core/src/components/Cards/SuccessStoryCard/SuccessStoryCardMobile.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+//@flow
+import * as React from 'react';
```
Comment (dbatten5) [inline packages/core/src/components/Cards/SuccessStoryCard/SuccessStoryCardMobile.js]: here too

File: packages/core/src/components/Cards/SuccessStoryCard/SuccessStoryCardMobile.js [hunk 3]
```diff
@@ -8,7 +8,17 @@ import RemoveOrphans from '../../RemoveOrphans/RemoveOrphans';
...
+  title: string,
+  imageSrc: string,
+  copy: string,
+  brands: Array<any>,
+  href: string,
+  variant?: string,
+  brandsTitle: string,
+}
+
+const SuccessStoryCardMobile = ({title, imageSrc, copy, brands = [], href, brandsTitle, variant}: Props) => (
```
Comment (dbatten5) [inline packages/core/src/components/Cards/SuccessStoryCard/SuccessStoryCardMobile.js]: for consistency is it worth giving brands a default value in `defaultProps` instead?

File: packages/core/src/components/Cards/DestinationListingCard/DestinationListingCard.js [hunk 4]
```diff
@@ -11,39 +11,38 @@ import HeartBtn from '../../HeartBtn/HeartBtn';
 import Icon from '../../Icon/Icon';
 import ScreenReadable from '../../ScreenReadable/ScreenReadable';
 import css from './DestinationListingCard.css';
+import type { Image } from '../../FittedImage/FittedImage';
```
Comment (Denyer) [inline packages/core/src/components/Cards/DestinationListingCard/DestinationListingCard.js]: this type doesn't exist

File: packages/core/src/components/Cards/EditorialCard/GuideCard.js [hunk 5]
```diff
@@ -1,7 +1,7 @@
-import PropTypes from 'prop-types';
-import React, { Component } from 'react';
+import * as React from 'react';
+
 import cx from 'classnames';
-import { applyContainerQuery } from 'react-container-query';
+import { applyContainerQuery, ContainerQuery } from 'react-container-query';
```
Comment (Denyer) [inline packages/core/src/components/Cards/EditorialCard/GuideCard.js]: ContainerQuery is unused

File: packages/core/src/components/Cards/CondensedSpaceCard/CondensedSpaceCard.js (hunk 1 above)
Comment (lukeamarsh) [reply packages/core/src/components/Cards/CondensedSpaceCard/CondensedSpaceCard.js]: yep. this a flow thing

File: packages/core/src/components/Cards/SuccessStoryCard/SuccessStoryCardMobile.js (hunk 2 above)
Comment (lukeamarsh) [reply packages/core/src/components/Cards/SuccessStoryCard/SuccessStoryCardMobile.js]: yeah we do unfortunately

File: packages/core/src/components/Cards/PictureCard/PictureCard.js [hunk 6]
```diff
@@ -1,16 +1,38 @@
...
+import * as React from 'react';
 import cx from 'classnames';
 
 import css from './PictureCard.css';
 import Card from '../Card/Card';
 
-const PictureCard = props => {
-  const { src, href, children, className, center, bottom, style, overlayClassName, ...rest } = props;
+type Props = {
+  src?: string,
```
[the remark this answers is not in the document]
Comment (lukeamarsh) [reply packages/core/src/components/Cards/PictureCard/PictureCard.js]: It isn't actually required 🙈

File: packages/core/src/components/Cards/PictureCard/PictureCard.js [hunk 7]
```diff
@@ -21,7 +43,7 @@ const PictureCard = props => {
       )}
       style={{
         ...style,
-        backgroundImage: `url(${src})`,
+        backgroundImage: src && `url(${src})`,
```
[the remark this answers is not in the document]
Comment (lukeamarsh) [reply packages/core/src/components/Cards/PictureCard/PictureCard.js]: it isn't required at all!

File: packages/core/src/components/Cards/AnimatedPictureCard/AnimatedPictureCard.js [hunk 8]
```diff
@@ -1,16 +1,22 @@
-import PropTypes from 'prop-types';
+//@flow
```
Comment (svobom57) [inline packages/core/src/components/Cards/AnimatedPictureCard/AnimatedPictureCard.js]: It's nice having Flow, but I think ESLint would complain about not having the space in between `//` and `@flow` 🤣

**Pull request merged**